### ISSUE TITLE
Bug in calculating error

### ISSUE THREAD

**Issue by jamesal1** [rythei/PyTorchOT#2]
Line 54 of ot_pytorch.py is
err = (torch.sum(transp) - b).norm(1).pow(2).data[0]

but should probably be
err = (torch.sum(transp,dim=0) - b).norm(2).pow(2).data[0]

i.e. the 2-norm should be applied, and a column sum instead of matrix sum should be calculated.

**Comment by dgrosbard11** [comment]
Yes, I have noticed this too.
